### PR TITLE
utility::loadCgltfBuffers is done once instead of doing for each primitive

### PR DESCRIPTION
**Pull request by mbalajee**
Fixes https://github.com/google/filament/issues/7930

### CODE REVIEW

**Comment by mbalajee** [comment]
@pirlande-fw

File: libs/gltfio/src/extended/AssetLoaderExtended.h
Comment (poweifeng) [inline libs/gltfio/src/extended/AssetLoaderExtended.h:86]: suggest `mCgltfBuffersLoaded`

**Review by poweifeng — APPROVED** [review]

**Comment by poweifeng** [comment]
> Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> View this [failed invocation](https://github.com/google/filament/pull/7969/checks?check_run_id=27241154886) of the CLA check for more information.
> 
> For the most up to date status, view the checks section at the bottom of the pull request.

Please sign the CLA

**Comment by poweifeng** [comment]
Thanks for fixing this.

**Comment by mbalajee** [comment]
@pixelflinger can you please review this?  i think PR is waiting for a maintainer's approval to get merged.  This is a blocker for our release, would be easier to get it from this repo instead of our fork.  Thanks

**Review by bejado — APPROVED** [review]

**Pull request merged**
